content/generalInformation.de.md

```diff
@@ -30,3 +30,8 @@ Bitte wende dich für die Beantragung und Inanspruchnahme an deine Personalabtei
 Wir sind ein engagiertes Team von Bahn-Mitarbeitenden, welche selbst regelmäßig FIP nutzen.
 Unser Ziel ist, eine schnell und einfach nutzbare Übersicht zur Nutzung von FIP bereitzustellen.
 Unser Fokus liegt auf der Bereitstellung von nützlichen Informationen für die Nutzerinnen und Nutzer.
+
+## Zusätzliche nützliche Ressourcen
+
+- Die [Rail Delivery Group](https://www.raildeliverygroup.com/rst/europe-and-fip.html) bietet eine ausführliche Übersicht zu FIP in verschiedenen Ländern.
+- Auf [fipcard.eu](https://fipcard.eu/) sind Informationen zu FIP in Frankfreich, Spanien, Deutschland, Finnland, Belgien, Litauen, Niederlande, Portugal, Österreich und Italien zu finden.
```

Comment:
Because the author of fipcard.eu isn't intrested in a cooperation or working together, I would prefere not linking the site.

Reply:
Ok, thanks for the information. I'll remove the mention for now.